src/tools/schema2kotlin.ts

```diff
@@ -40,14 +40,20 @@ export class Schema2Kotlin extends Schema2Base {
   }
 
   fileHeader(outName: string): string {
+    const withCustomPackage = (populate: string) => this.scope !== 'arcs' ? populate : '';
     return `\
 package ${this.scope}
 
 //
 // GENERATED CODE -- DO NOT EDIT
 //
 // Current implementation doesn't support references or optional field detection
-`;
+
+${withCustomPackage(`import arcs.Particle;
```

Comment:
Do we have unit tests for `schema2kotlin.ts`? If not perhaps we should start some, and add test cases to cover the new logic added here and below.

Reply:
Great question. These are on the docket (Kotlin entity generation tests). Right now, I'm writing integration tests for the particle API. This PR is somewhat of a blocker for the integration tests.

Reply:
A blocker in the sense that it will make having a clean revision history easier.

Reply:
@shaper: Alex had added some tests for schema2kotlin/cpp, but they were made obsolete by the refactor I did for the type lattice (generally replaced by tests covering similar logic in the graph builder). I do want to go back and add more language-specific tests on the generated output at some point.